app/routes.py

```
@@ -792,7 +824,7 @@ async def meetingbaas_webhook(request: Request):
         return {"status": "error", "message": str(e)}
 
 
-async def generate_summary_from_transcript(transcript_file: str, bot_id: str = None):
+def _generate_summary_sync(transcript_file: str, bot_id: str = None):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_generate_summary_sync`

Add return type annotation: `None`

(ANN202)